low-level/files/opt/seagate/sspl/bin/feed_sspl_conf_to_consul.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2001-2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+"""
+ ******************************************************************************
+  Description:   Parse config file and arguments to set common & SSPL config
+  Purpose    :   To feed sspl and common configuration to consul
+  Usage      :   python3 feed_sspl_conf_to_consul.py [-F] [-C] [-N] [-Ru] [-Rp]
+ ******************************************************************************
+"""
+
+import re
+import os
+import sys
+import consul
+import argparse
+from configobj import ConfigObj
```

Comment:
Don't introduce new pkg dependency until its must. And if its must, needs to get reviewed with ARC team.
configparser package imported already serves the purpose, why the need to include new ConfigObj now ?

Reply:
I used this for reading/adding nested config in ini file. Now, its removed as we just okay with sspl.conf itself.
I removed ConfigObj now.